glyphs2fontir/src/glyphdata.rs

```diff
@@ -4,8 +4,705 @@
 
 use std::collections::HashSet;
 
-pub(crate) fn is_nonspacing_mark(codepoints: &HashSet<u32>, name: &str) -> bool {
-    // Try first by name
+#[inline(always)]
+pub(crate) fn might_be_a_nonspacing_mark_name(name: &str) -> bool {
+    // shove the first 4 chars into a u32 and see if they could possibly be a nonspacing mark
+    let head = name
+        .chars()
+        .enumerate()
+        .take(4)
+        .fold(0u32, |acc, (i, ch)| acc | (ch as u32) << (i * 8));
```

Comment:
we're taking chars but then truncating them, could we just use `bytes()`?

Reply:
Yes.